### DISCUSSION TITLE
Permission denied for client

### DISCUSSION THREAD

**Discussion by Romain-Fromantin** [l7mp/stunner#130] [Q&A]
Hello,
Is the turn server blocking or it cannot create the connection?
`turn.go:239: turn INFO: permission denied for client 192.168.1.182:57500 to peer 192.168.1.57`
192.168.1.182:57500: One of the nodes of the Kubernetes cluster 
192.168.1.57: IP du client

**Comment by rg0now** [comment]
"Permission denied" errors mean that the client tries to reach a peer via a UDPRoute that it has no permission to reach. This usually means that you forgot to include peer (this time, it's the pod with IP `192.168.1.57`) in the Kubernetes Service that is set as a backend in the UDPRoute.
